docs/blender_docs/scene_gltf2.rst

```diff
@@ -52,10 +54,14 @@ with the following channels of information:
 - Metallic
 - Roughness
 - Baked Ambient Occlusion
-- Normal Map
+- Normal Map (Tangent-space, +Y up)
 - Emissive
-- Clearcoat (requires ``KHR_materials_clearcoat``)
-- Transmission (requires ``KHR_materials_transmission``)
+
+Some additional material properties or types of materials can be expressed using glTF extensions:
+
+- Clearcoat (uses ``KHR_materials_clearcoat``)
```

Comment:
Should we include Clearcoat Roughness and Clearcoat Normal Map here?

Reply:
Done